### DISCUSSION TITLE
Error on adding workflow_call:

### DISCUSSION THREAD

**Discussion by metabsd** [BretFisher/super-linter-workflow#2] [Q&A] [answered]
You know why ?

![image](https://user-images.githubusercontent.com/7953470/135612486-a96de787-def3-420f-ab57-cf9497fd405a.png)

Thx!

**Comment by BretFisher** [comment, marked as answer]
[actionlint](https://github.com/rhysd/actionlint) likely needs to update their rules to understand the new beta reusable workflows feature of GitHub Actions.

**Reply by metabsd** [reply]
Thx a lot for sharing that. I really liked your youtube channel.
